<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Widget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_plus">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus_minus">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>40</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_size">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_size">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_size">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="equal">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_move">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_move">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_move">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="move_go">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>110</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_turn">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>150</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_turn">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>150</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>180</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>turn</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_turn">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>150</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="restart">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>490</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget</class>
   <extends>QWidget</extends>
   <header>widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
